### DISCUSSION TITLE
挑战NFT有中英文，那元数据使用中文还是英文？

### DISCUSSION THREAD

**Discussion by ccxdcyl** [decert-me/website#299] [General]
遇到的情况：
现在DeCert每个挑战都有对应的中/英文，当用户在中文状态下完成挑战，然后用户把网站切换成英文，此时去领取，元数据是使用中文还是英文

目前的两个方案：
1.在领取时，按照用户提交挑战时的语言去铸造
2.无论中/英文，在铸造时，将中英文都写入元数据内

**Comment by ccxdcyl** [comment]
支持方案【1】

**Comment by xilibi2003** [comment]
元数据可以根据用户当前选择的语言，把对应语言写入到 元数据中。

挑战内容本身可以有多种语言。
